packages/teams-js/test/public/location.spec.ts

```diff
@@ -326,4 +326,66 @@ describe('location', () => {
       });
     });
   });
+
+  it('location.isSupported should return false if the runtime says location is not supported', () => {
+    framelessPlatform.setRuntimeConfig({ apiVersion: 1, supports: {} });
+    expect(location.isSupported()).not.toBeTruthy();
+  });
+
+  it('location.isSupported should return true if the runtime says location is supported', () => {
+    framelessPlatform.setRuntimeConfig({ apiVersion: 1, supports: { location: {} } });
+    expect(location.isSupported()).toBeTruthy();
+  });
+
+  it('Frameless - getLocation should throw error when not supported in the runtime config', () => {
+    framelessPlatform.setRuntimeConfig({ apiVersion: 1, supports: {} });
+    framelessPlatform.initializeWithContext(FrameContexts.task).then(() => {
+      expect.assertions(4);
+
+      try {
+        location.getLocation(defaultLocationProps, emptyCallback);
+      } catch (e) {
+        expect(e).toEqual(errorNotSupportedOnPlatform);
+      }
+    });
+  });
+
+  it('Frameless - showLocation should throw error when location is not supported', async () => {
+    framelessPlatform.setRuntimeConfig({ apiVersion: 1, supports: {} });
+    framelessPlatform.initializeWithContext(FrameContexts.task).then(() => {
+      expect.assertions(4);
+
+      try {
+        location.showLocation(defaultLocation, emptyCallback);
+      } catch (e) {
+        expect(e).toEqual(errorNotSupportedOnPlatform);
+      }
+    });
+  });
+
+  it('Framed - getLocation should throw error when not supported in the runtime config', () => {
+    framelessPlatform.setRuntimeConfig({ apiVersion: 1, supports: {} });
```

Comment:
This says Framed, but you're still using the framelessPlatform mock inside the test.

Reply:
Thanks for catching it!